src/server/process_request.c

```diff
@@ -843,6 +843,11 @@ dispatch_request(int sfds, struct batch_request *request)
 			break;
 
 		case PBS_BATCH_DeleteJobList:
+			log_event(PBSEVENT_DEBUG, PBS_EVENTCLASS_JOB, LOG_INFO,
+				request->rq_ind.rq_deletejoblist.rq_jobslist[0],
+				"delete job request received");
```

Comment:
add the word 'list' here to differentiate it from a regular deljob

Reply:
Some test cases depend on this log msg. I think, for both the delete and deletejoblist request this log msg would be appropriate. If you feel otherwise, Let me know, I will change this log msg and its corresponding test cases.